deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
+repo=${2-LSSTDESC/start_paper}
+branch=${3-master}
```

Comment:
This script seems more general than we need: as far as I can see, it is only used to start papers with, and so could be streamlined to remove argument 2 I think. I was expecting to download a script called `start_paper.bash` which could live in my bin directory and be summoned by me typing "start" and then hitting the tab and return keys. And then, I was not expecting to need to tell the script which repo to use! I just want to start a paper. What do you think?